_data/meltano/extractors/tap-db2/danielptv.yml

```
@@ -0,0 +1,72 @@
+capabilities:
+- about
+- catalog
+- discover
+- state
+- stream-maps
+description: "Tap-DB2 is a Singer tap for IBM DB2 data sources."
+domain_url: https://github.com/danielptv/tap-db2
```

```suggestion
domain_url: https://www.ibm.com/products/db2
```

Similar question to https://github.com/meltano/hub/pull/1510/files#r1320208110